### PR TITLE
GraphDefinition for Test Result

### PR DESCRIPTION
**Pull request by omarisgreat**
also includes minor changes to test order GraphDefinition and DocumentReference (based on updated IG guidance)

### CODE REVIEW

File: DocumentReference/DocumentReference-PharmCAT-Example.json
Comment (KazeemHamzat) [inline DocumentReference/DocumentReference-PharmCAT-Example.json:1]: As the DocumentReference resource definition in R4 has no version element. Will the date created just be changing to any time it's modified?

Comment (omarisgreat) [reply DocumentReference/DocumentReference-PharmCAT-Example.json:1]: I expect the version and date modified would instead be captured under the meta element. I don't think there is any use case for modifying this resource after creation however, as the drs link is a permanent pointer. (and any new data files should result in new DocumentReference resources)

File: GraphDefinition/GraphDefinition-Genomics-Test-Result.json [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+  "url": "https://fhir.nhs.uk/GraphDefinition/genomics-test-result",
+  "name": "GenomicsTestResult",
+  "status": "draft",
+  "date": "2024-08-09",
+  "publisher" : "NHS England",
+  "description": "GraphDefinition for defining and retrieving the full set of resources relevant to a Genomic Test Result (based on the Genomics Reporting Genomic-Report profile)",
+  "start": "DiagnosticReport",
+  "link": [
+    {
+      "path": "DiagnosticReport.extension.valueReference",
```
Comment (KazeemHamzat) [inline GraphDefinition/GraphDefinition-Genomics-Test-Result.json:13]: Not sure if it needs to exist on UKCore or NHSD DiagnosticReport profile but I can't find the extension name, "sliceName": "genomic-study" on either.

File: GraphDefinition/GraphDefinition-Genomics-Test-Result.json [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+      "description": "The set of analyses performed to generate/analyse the genomic data",
+      "min": 0,
+      "max": "*",
+      "target": [
+        {
+          "type": "Procedure",
+          "link": [
+            {
+              "path": "Procedure.extension.valueReference",
+              "sliceName": "genomic-study-analysis",
```
Comment (KazeemHamzat) [inline GraphDefinition/GraphDefinition-Genomics-Test-Result.json:36]: Same as above

File: CapabilityStatement/CapabilityStatement-genomics-application.json [hunk 3]
```diff
@@ -1,1792 +1,1871 @@
...
-						{
-							"name": "title",
-							"type": "string"
-						}
-					]
-				},
+  "resourceType": "CapabilityStatement",
+  "id": "CapabilityStatement-genomics-application",
+  "url": "https://fhir.nhs.uk/CapabilityStatement/CapabilityStatement-genomics-application",
+  "version": "0.1.0",
```
Comment (KazeemHamzat) [inline CapabilityStatement/CapabilityStatement-genomics-application.json:5]: version? 0..2.0, and date?

File: GraphDefinition/GraphDefinition-Genomics-Test-Order.json
Comment (KazeemHamzat) [inline GraphDefinition/GraphDefinition-Genomics-Test-Order.json:1]: Version, Title omitted and date and version will cahnge.

**Review by KazeemHamzat — APPROVED** [review]
Minor changes around meta. Looks good to me.

File: GraphDefinition/GraphDefinition-Genomics-Test-Result.json [hunk 4]
```diff
@@ -0,0 +1,181 @@
+{
+  "resourceType": "GraphDefinition",
+  "id": "GraphDefinition-Genomics-Test-Result",
```
Comment (KazeemHamzat) [inline GraphDefinition/GraphDefinition-Genomics-Test-Result.json:3]: For conformance to all other assets that have the ID as part of their canonical URL, the ID should be all lower case.

File: GraphDefinition/GraphDefinition-Genomics-Test-Result.json (hunk 1 above)
Comment (omarisgreat) [reply GraphDefinition/GraphDefinition-Genomics-Test-Result.json:13]: These are on the genomic reporting IG GenomicReport profile, this is pending further review before officially bringing it into the Genomics IG but this GraphDefinition is just a demonstration of how the extensions could be traversed (the slices should be ignored if they are not found on the DiagnosticReport)

**Pull request merged**